docs/en/query_language/system.md

```diff
@@ -11,6 +11,7 @@
 - [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
 - [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
 - [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
+- [STOP|START MERGES](#query_language-system-stop-start-merges)
```

Comment:
we'd better use separate descriptions for start and stop to make it consistent with "distributed sends" few lines above